### DISCUSSION TITLE
Where to find more Litestar examples for the intermediate developer?

### DISCUSSION THREAD

**Discussion by tomschr** [litestar-org/litestar#3513] [Q&A] [answered]
I really like Litestar so far! :heart: It's a awesome project, thanks for all the great work.

I'm currently learning Litestar, reading the documentation and trying out things like the simple simple ["Hello World"](https://github.com/litestar-org/litestar-hello-world). This was all good, but I feel I hit a wall.

I'd like to study an existing project that goes a bit beyond the simple "Hello World" example. It's a good start for the very beginner, but I think I know a little bit more now. On the other side, I'm not an expert. So I think, there is a gap for the _intermediate developer_.

To fill this void, I did:

* Looked in the Litestar organization and found the [litestar/litestar-fullstack](https://github.com/litestar-org/litestar-fullstack) project. Very impressive, but that's a bit too much for my taste.
* Followed the [`litestar-framework` tag](https://github.com/search?q=topic%3Alitestar-framework&type=repositories) on GitHub. It shows some plugins and mainly results from the Litestar org itself, but not any projects so far.
* Read the [Tutorial](https://docs.litestar.dev/latest/tutorials/index.html). It's very well written, but it seems there is no GitHub repo with the full code, including tests etc.

So I'm a bit lost. :shrug:  Maybe someone else has a link or a project that comes into your mind?

My ideal example would be something like this:

* Introduces an simple idea which is implemented in Litestar. Could be a blog or something else that is easy to understand.
* Has a decent size, not too big (as the litestar-fullstack example), but also not too small (like the litestar-hello-world)
* Applies best practices, follows 12 factor apps, is modular, and easily extensible.
* Contains a fully test suite

Thank you for your ideas!

**Comment by peterschutt** [comment, marked as answer]
This is a request that keeps popping up - discussion is underway about what this may look like and how it will materialize. We'll keep you posted.

**Reply by tomschr** [reply]
Thanks a lot and sorry for the noise.

Perhaps it would also help to amend the documentation with some best practices?
